src/drivers/ports/avr/adc_port.h

```diff
@@ -37,6 +37,7 @@ struct adc_driver_t {
     struct adc_device_t *dev_p;
     struct pin_driver_t pin_drv;
     uint8_t admux;
+    uint8_t adcsrb;
```

Comment:
Please add conditional compilation since smaller AVR:s do not need to modify ADCSRB, right?

```
#if defined(MUX5)
    uint8_t adcsrb;
#endif
```